source/adapters/cuda/kernel.cpp

```diff
@@ -299,6 +299,7 @@ UR_APIEXPORT ur_result_t UR_APICALL urKernelGetInfo(ur_kernel_handle_t hKernel,
                                       hKernel->get()));
     return ReturnValue(static_cast<uint32_t>(NumRegs));
   }
+  case UR_KERNEL_INFO_SPILL_MEM_SIZE:
```

Comment:
for here and in the equivalent file in the hip adapter you'll need to explicitly add a `return UR_RESULT_ERROR_UNSUPPORTED_ENUMERATION;` - right now it's falling through to `return UR_RESULT_ERROR_INVALID_ENUMERATION;` which isn't the expected return for an unsupported optional query

Reply:
done